### PR TITLE
Add de_DE.lang

### PR DESCRIPTION
**Pull request by Lignum**
Not sure what branch this fits in, I just chose the default.
Translation should be mostly accurate, though I'm not entirely sure as I haven't used some features yet.

### CODE REVIEW

File: src/main/resources/assets/plethora/lang/de_DE.lang [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+# Gameplay config
+gui.config.plethora.laser=Verdammter Laserstrahl
+gui.config.plethora.laser.laserMinimum=Laser Minimum
+gui.config.plethora.laser.laserMaximum=Laser Maximum
+gui.config.plethora.laser.laserCost=Laserpreis
+
+gui.config.plethora.kinetic=Kinetischer Verstärker
+gui.config.plethora.kinetic.launchMax=Maximaler Schleuderfaktor
+gui.config.plethora.kinetic.launchCost=Maximaler Schleuderpreis
```
Comment (Lignum) [inline src/main/resources/assets/plethora/lang/de_DE.lang:69]: launchMax and launchCost are both "launch factor" in en_US.lang, I translated the latter as "launch cost" here, you might wanna check that.

File: src/main/resources/assets/plethora/lang/de_DE.lang [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+gui.config.plethora.kinetic.walkRange=Maximale Laufreichweite
+gui.config.plethora.kinetic.walkSpeed=Maximale Laufgeschwindigkeit
+gui.config.plethora.kinetic.walkCost=Laufkosten
+gui.config.plethora.kinetic.teleportRange=Maximale Teleportationsreichweite
+gui.config.plethora.kinetic.teleportCost=Teleportationspreis
+
+gui.config.plethora.scanner=Entityscanner
```
Comment (Lignum) [inline src/main/resources/assets/plethora/lang/de_DE.lang]: Both plethora.scanner and plethora.sensor are "Entity Sensor" in en_US.lang, that also seems wrong.

**Comment by SquidDev** [comment]
Sorry for taking so long to get to this. Good catch on those incorrect English translations. 

Do you think you could add a line for `gui.config.plethora.kinetic.launchElytraScale`? This is only in the 1.10 version but it is probably easier to keep all translation data on the 1.8.9 branch.

---

**Edit:** Actually, can you include [all of these additional lines/changes](https://github.com/SquidDev-CC/plethora/commit/79e1628b067edb244a8995006ecbddb570c9f76d#diff-c3784c7c3b49785340e57b222878782a).

**Comment by Lignum** [comment]
@SquidDev All done!

**Comment by SquidDev** [comment]
@Lignumm Ahhh, sorry. I [added a couple more entries](https://github.com/SquidDev-CC/plethora/commit/79e1628b067edb244a8995006ecbddb570c9f76d#diff-c3784c7c3b49785340e57b222878782a). Do you think you could do those too? Sorry 😦.

**Comment by Lignum** [comment]
@SquidDev Since you removed some localisations, I wasn't sure which ones were still needed in 1.8. So, remove them as you please.

**Comment by SquidDev** [comment]
Great, thank you very much!

**Comment by Lignum** [comment]
Cheers :tada:

**Pull request merged**